core/remotes/docker/fetcher.go

```diff
@@ -262,7 +265,31 @@ func (r dockerFetcher) FetchByDigest(ctx context.Context, dgst digest.Digest, op
 	return seeker, desc, nil
 }
 
+const (
+	_        = iota
+	kB int64 = 1 << (10 * iota)
+	mB
+	gB
+	tB
+	pB
+	eB
+)
+
 func (r dockerFetcher) open(ctx context.Context, req *request, mediatype string, offset int64) (_ io.ReadCloser, retErr error) {
+	parallelism := int64(1)
```

Comment:
Nit: if parallelism = 1 is the default behavior, it may be useful to define this value as constants  in the beginninga long with a coment.

Reply:
Good idea, adding this.